source-chargebee-native/source_chargebee_native/resources.py

```diff
@@ -34,92 +39,136 @@
     snapshot_resource,
     fetch_resource_page,
     fetch_resource_changes,
+    snapshot_associated_resource,
+    fetch_associated_resource_page,
+    fetch_associated_resource_changes,
 )
 
 
-# TODO(jsmith): revisit
-# PARENT_CHILD_RESOURCES: dict[str, tuple[str, str]] = {
-#     "contacts": ("customers", "id"),  # (parent_resource, parent_key_field)
-#     "subscriptions_with_scheduled_changes": ("subscriptions", "id"),
-# }
+COMMON_STANDALONE_FULL_REFRESH_RESOURCES = [
+    "site_migration_details",
+    "quote_line_groups",  # Requires Performance or Enterprise Chargebee subscription plan - will throw HTTP 404 if not enabled
+]
 
-STANDALONE_FULL_REFRESH_RESOURCES: dict[Literal["1.0", "2.0"], list[str]] = {
+FULL_REFRESH_RESOURCES: dict[Literal["1.0", "2.0"], list[str]] = {
     "1.0": [
-        # "comments", # TODO(jsmith): revisit - potentially incremental
-        # "gifts", # TODO(jsmith): revisit - potential client-side filtering
-        # "promotional_credits", # TODO(jsmith): revisit - potentially incremental
-        "site_migration_details",
-        # "unbilled_charges", # TODO(jsmith): revisit - potential client-side filtering
-        # "quote_line_groups", # TODO(jsmith): add this back in after figuring out the best way to handle 404 (quotes feature not enabled)
+        *COMMON_STANDALONE_FULL_REFRESH_RESOURCES,
+        "gifts",
+        "unbilled_charges",
     ],
     "2.0": [
-        # "comments", # TODO(jsmith): revisit - potentially incremental
-        # "differential_prices", # TODO(jsmith): revisit
-        # "gifts", # TODO(jsmith): revisit - potential client-side filtering
-        # "promotional_credits", # TODO(jsmith): revisit - potentially incremental
-        "site_migration_details",
-        # "unbilled_charges", # TODO(jsmith): revisit - potential client-side filtering
-        # "quote_line_groups", # TODO(jsmith): add this back in after figuring out the best way to handle 404 (quotes feature not enabled)
+        *COMMON_STANDALONE_FULL_REFRESH_RESOURCES,
+        "differential_prices",
```

Comment:
The only difference I see between the 1.0 and 2.0 full refresh resources is that 2.0 has `differential_prices`, so maybe this could be simplified?